r/tests/testthat/test-dataset.R

```diff
@@ -98,6 +98,11 @@ test_that("Simple interface for datasets", {
   )
 })
 
+test_that("dim method returns the correct number of rows and columns",{
+  ds <- open_dataset(dataset_dir, partitioning = schema(part = uint8()))
+  expect_equal(dim(ds), c(20, 7))
```

Comment:
This should be `expect_identical` as you previously had it: we want to make sure that any code that expects `dim()` to return a certain shape and type is also valid for Datasets. I suspect your test was failing because you did `map_dbl` instead of `map_int`.

Reply:
thanks for catching this